skycultures/arabian_peninsula/constellation_names.eng.fab

```diff
@@ -0,0 +1,40 @@
+# TRANSLATORS: Aba Bzay is a proper name
+100 "الحويجزين - الدخيلين - آل أبا بزي" _("The Two Guards - The Two Refugees - The Family Of Aba Bzay")

```

Comment:
out of curiosity: is the ID a number code? Object list in some book?

Reply:
No.
I built these sky culture along al-Sufi sky culture beginning with northern constellations. The numbers used are just to facilitate later additions if any, in the same or close by regions. Using abbreviations here would be confusing.
I published two books on this subject, but I didn't have tables or a catalog of folk names.